src/main/io/spektrum_vtx_control.c

```diff
@@ -0,0 +1,188 @@
+//#define USE_VTX_COMMON_FREQ_API
+
+#ifdef USE_VTX_COMMON_FREQ_API
+    const uint16_t SpektrumVtxfrequencyTable[SPEKTRUM_VTX_BAND_COUNT][SPEKTRUM_VTX_CHAN_COUNT] =
+      {
+        { 5740, 5760, 5780, 5800, 5820, 5840, 5860, 5880 }, // FatShark
+        { 5658, 5695, 5732, 5769, 5806, 5843, 5880, 5917 }, // RaceBand
+        { 5705, 5685, 5665, 5645, 5885, 5905, 5925, 5945 }, // Boscam E
+        { 5733, 5752, 5771, 5790, 5809, 5828, 5847, 5866 }, // Boscam B
+        { 5865, 5845, 5825, 5805, 5785, 5765, 5745, 5725 }, // Boscam A
+      };
+#else
+    // Translation table, Spektrum bands to BF internal vtx_common bands
+    const uint8_t spek2commonBand[]= {
+      VTX_COMMON_BAND_FS,
+      VTX_COMMON_BAND_RACE,
+      VTX_COMMON_BAND_E,
+      VTX_COMMON_BAND_B,
+      VTX_COMMON_BAND_A,
+    };
+#endif
+
+    // RF Power Index translation tables. No generic power API available.....
+
+    // Tramp "---", 25, 200, 400. 600 mW
+const uint8_t vtxTrampPi[] = {           // Spektrum Spec    Tx menu  Tx sends   To VTX    Wand
+      VTX_TRAMP_POWER_OFF,               //         Off      INHIBIT         0        0     -
+      VTX_TRAMP_POWER_OFF,               //   1 -  14mW            -         -        -     -
+      VTX_TRAMP_POWER_25,                //  15 -  25mW   15 -  25mW         2        1    25mW
+      VTX_TRAMP_POWER_100,               //  26 -  99mW   26 -  99mW         3        2   100mW Slightly outside range
+      VTX_TRAMP_POWER_200,               // 100 - 299mW  100 - 200mW         4        3   200mW
+      VTX_TRAMP_POWER_400,               // 300 - 600mW  300 - 600mW         5        4   400mW
+      VTX_TRAMP_POWER_600,               // 601 - max    601+ mW             6        5   600mW Slightly outside range
+      VTX_TRAMP_POWER_200                // Manual               -           -        -     -
+    };
+
+    // RTC6705 "---", 25 or 200 mW
+    const uint8_t vtxRTC6705Pi[] = {
+      VTX_6705_POWER_OFF,                // Off
+      VTX_6705_POWER_OFF,                //   1 -  14mW
+      VTX_6705_POWER_25,                 //  15 -  25mW
+      VTX_6705_POWER_25,                 //  26 -  99mW
+      VTX_6705_POWER_200,                // 100 - 299mW
+      VTX_6705_POWER_200,                // 300 - 600mW
+      VTX_6705_POWER_200,                // 601 - max
+      VTX_6705_POWER_200                 // Manual
+    };
+
+    // SmartAudio "---", 25, 200, 500. 800 mW
+    const uint8_t vtxSaPi[] = {
+      VTX_SA_POWER_OFF,                  // Off
+      VTX_SA_POWER_OFF,                  //   1 -  14mW
+      VTX_SA_POWER_25,                   //  15 -  25mW
+      VTX_SA_POWER_25,                   //  26 -  99mW
+      VTX_SA_POWER_200,                  // 100 - 299mW
+      VTX_SA_POWER_500,                  // 300 - 600mW
+      VTX_SA_POWER_800,                  // 601 - max
+      VTX_SA_POWER_200                   // Manual
+    };
+
+    uint8_t convertSpektrumVtxPowerIndex(uint8_t sPower)
```

Comment:
Can you change to standard indenting, ie 4 spaces. And code under `#define`s is not shifted right.

Reply:
OK. Will check and change.